content/en/docs/how-tos/cluster-claim.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Creating a Cluster Pool"
+description: How to create a cluster pool from which a job can claim a cluster.
+---
+
+If a CI user wants to use their own cloud account to provision clusters, the steps of creating such a cluster pool can be found in this page.
+
+## Prerequisites
+
+* Vefirify that the cloud platform you want to use is supported by [Hive](https://github.com/openshift/hive#supported-cloud-providers) which is the backbone of cluster pools in the test platform.
+* [Configure the cloud account](https://docs.openshift.com/container-platform/4.7/installing/installing-preparing.html): Hive does not require more configuration of the cloud account than installing an OpenShift cluster. Make sure that the account has quotas for the pools to contain the desired number of clusters.
+
+## Create a cluster pool
+
+Create a folder in `clusters/app.ci/pools` for the manifests of the pool. See [openshift-ci](https://github.com/openshift/release/tree/master/clusters/app.ci/pools/openshift-ci) as an example for the existing `ci-cluster-pool`.
+
+```yaml
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: ci-ocp-4-7-amd64-aws-us-east-1
+  namespace: cvp-cluster-pool
+  labels: # product, version, architecture, cloud, owner are used to filter out a pool when a job claims a cluster
+    product: ocp
+    version: "4.7"
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    region: us-east-1
+spec:
+  baseDomain: hive.example.org # the base domain to install the cluster
+  imageSetRef:
+    name: ocp-4.7.0-amd64 # the name of the imageSet which determines the image to install the cluster
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1 # the name of the secret with an installation config for the installer
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: hive-aws-credentials # the name of the secret with the credentials of the cloud account
+      region: us-east-1
+  pullSecretRef:
+    name: pull-secret # the name of the pull secret used by installer
+  size: 1 # the number of clusters that Hive should keep provisioned and waiting for use.
+  maxSize: 10 # the maximum number of clusters that have not been destroyed.
+```
```

Comment:
This only describes one of the necessary resources, but the existing folder contains more - a secret used as an install config, a clusterimageset, a namespace? The instructions should discuss them.

Reply:
Updated.